<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellDossPanel</class>
 <widget class="QDialog" name="SellDossPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>256</width>
     <height>213</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Dossier ID: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>House Type: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>ID: [Apartment]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Commission: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Sell Condition: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Final Price: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lDID"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lHType"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lID"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lCommission"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lSCond"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lFPrice"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pRequest">
        <property name="text">
         <string>Request</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pHouse">
        <property name="text">
         <string>House</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
